megamek/src/megamek/common/Bay.java

```diff
@@ -79,19 +81,42 @@ public Bay(double space, int doors, int bayNumber) {
         totalSpace = space;
         currentSpace = space;
         this.doors = doors;
-        doorsNext = doors;
+        doorsNext = currentdoors;
         this.bayNumber = bayNumber;
     }
-
+    
+    // For tracking damage to the bay itself, not the cargo it carries
+    public int getbayDamaged() {
```

Comment:
B in bay should be capitalized.  The other methods relating to bay damage should also be updated.

Reply:
Fixed.